uds/transport_interface/python_can_transport_interface.py

```diff
@@ -0,0 +1,178 @@
+"""Implementation of Transport Interfaces for CAN bus handled by python-can."""
+
+__all__ = ["PyCanTransportInterface"]
+
+from typing import Optional, Any
+
+from can import BusABC
+
+from uds.utilities import TimeMilliseconds
+from uds.packet import CanPacket, CanPacketRecord
+from uds.message import UdsMessage, UdsMessageRecord
+from uds.can import CanAddressingFormatAlias
+from uds.segmentation import CanAIArgsAlias
+from .abstract_can_transport_interface import AbstractCanTransportInterface
+
+
+class PyCanTransportInterface(AbstractCanTransportInterface):
+    """
+    Transport Interface for CAN that is compatible with python-can.
+
+    Documentation for python-can: https://python-can.readthedocs.io/
+    """
+
+    def __init__(self,  # pylint: disable=super-init-not-called
```

Comment:
`__init__` is probably obsolete (I think `AbstractCanTransportInterface` can manage it)